sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/Models/PostgreSqlFlexibleServerVirtualNetworkSubnetUsageResult.Serialization.cs

```diff
@@ -15,43 +15,43 @@ public partial class PostgreSqlFlexibleServerVirtualNetworkSubnetUsageResult
     {
         internal static PostgreSqlFlexibleServerVirtualNetworkSubnetUsageResult DeserializePostgreSqlFlexibleServerVirtualNetworkSubnetUsageResult(JsonElement element)
         {
+            Optional<IReadOnlyList<PostgreSqlFlexibleServerDelegatedSubnetUsage>> delegatedSubnetsUsage = default;
             Optional<AzureLocation> location = default;
             Optional<string> subscriptionId = default;
-            Optional<IReadOnlyList<PostgreSqlFlexibleServerDelegatedSubnetUsage>> delegatedSubnetsUsage = default;
```

Comment:
Same breaking change. Not an issue but it's better to avoid.

Reply:
Need to resolve since there is already an official version.

Reply:
I only see order changes here, this is not a breaking change. The API file in this RP shows this PR only introduces additions